### ISSUE TITLE
Problem: Thumbnails directory (sharedDirectory/www/thumbnails) is not cleared on AIP/DIP completion

### ISSUE THREAD

**Issue by ross-spencer** [archivematica/Issues#1009]
**Expected behaviour**

Archivematica storage does two things:

1. Maintain state between microservice jobs.
2. Store AIPs/SIPs/DIPs/AICs.

In the case of 1. state should be wiped as much as is feasible once a package has been generated. 

**Current behaviour**

In the case of generating preview thumbnails, they are stored in `sharedDirectory/www/thumbnails` and never removed. This space can build up. The example seen today:

```
root@archivematica:/var/archivematica/sharedDirectory/www# ls -la thumbnails/ | wc -l
240
```

```
du -h thumbnails
142M	thumbnails/
```

We don't provide users with a mechanism to clear this space on the `Processing storage usage` tab. Admin needs to go into the command line to wipe it. 

I wonder too if `www` is not the correct folder for storing these items? (unless we expect them to be served up to the user from here?)

**Steps to reproduce**

1. Configure AM to normalize for access.
2. Configure AM to generate thumbnails.
3. Transfer some images and create a DIP.

Observe the shared directory location described above on completion. 

**Your environment (version of Archivematica, operating system, other relevant details)**

Default configuration for ACDPS. Verified on Docker running `qa/1.x`. 

---

**For Artefactual use:**

Before you close this issue, you must check off the following:

- [ ] All pull requests related to this issue are properly linked
- [ ] All pull requests related to this issue have been merged
- [ ] A testing plan for this issue has been implemented and passed (testing plan information should be included in the issue body or comments)
- [ ] Documentation regarding this issue has been written and merged (if applicable)
- [ ] Details about this issue have been added to the release notes (if applicable)

**Comment by sromkey** [comment]
Does the Archival Storage tab use the thumbnails in this directory or do they come from elsewhere? Anyone know?

**Comment by ross-spencer** [comment]
@sromkey I can confirmthat this is exactly what it is used for so I can close this now as I have logged it in error. 

To test I did the following:

1. Created a transfer with a cat image. 
2. Normalized for preservation and access and created Thumbnails. 
3. Searched for the file in Archival Storage with `show files`:

![image](https://user-images.githubusercontent.com/1880412/72918366-87c92200-3d13-11ea-831b-02d3e29ec153.png)

4. I then accessed the storage service and saw the following layout:

```
root@d9f5de3ccd72:/var/archivematica/sharedDirectory# tree www/thumbnails/
www/thumbnails/
└── 33a326e2-6942-4b36-b3eb-30a3ae710917
    └── d6f91054-2eec-427a-99c4-db9abb984bdb.jpg
```

5. I then replaced `d6f91054-2eec-427a-99c4-db9abb984bdb.jpg` with my own custom image and performed the same file search:

![image](https://user-images.githubusercontent.com/1880412/72918606-e8f0f580-3d13-11ea-8b7b-2dd7d692b5a5.png)

So this new image was served to the end-user from the `www/thumbnails/` directory.
